py_inspect.py

```diff
@@ -173,154 +177,175 @@ def __init__(self):
-            'is_collapsed': None,
-            'get_selection': None,
-            'get_show_state': None,
-            'get_expand_state': None,
-            'select': None
+        self.backend_methods = {
+            'win32': {
+                # element_info_win = pywinauto.backend.registry.backends['win32'].element_info_class()
+                # print(set([attr for attr in dir(pywinauto.controls.hwndwrapper.HwndWrapper) if not attr.startswith('_') and not attr[0].isupper() and inspect.ismethod(getattr(pywinauto.controls.hwndwrapper.HwndWrapper(element_info_win), attr))]))
+                'backend': {
+                    'full_control_type': None,
+                    'set_keyboard_focus': None,
+                    'style': None,
+                    'set_transparency': None,
+                    'context_help_id': None,
+                    'popup_window': None,
+                    'has_exstyle': None,
+                    'control_type': None,
+                    'maximize': None,
+                    'is_minimized': None,
+                    'has_focus': None,
+                    'fonts': None,
+                    'owner': None,
+                    'double_click': None,
+                    'font': None,
+                    'send_command': None,
+                    'menu_item': None,
+                    'send_message': None,
+                    'menu_select': None,
+                    'handle': None,
+                    'client_rect': None,
+                    'set_application_data': None,
+                    'click': None,
+                    'right_click': None,
+                    'client_rects': None,
+                    'send_keystrokes': None,
+                    'debug_message': None,
+                    'press_mouse': None,
+                    'has_keyboard_focus': None,
+                    'post_message': None,
+                    'move_window': None,
+                    'minimize': None,
+                    'is_unicode': None,
+                    'set_window_text': None,
+                    'post_command': None,
+                    'exstyle': None,
+                    'scroll': None,
+                    'get_toolbar': None,
+                    'get_focus': None,
+                    'is_normal': None,
+                    'menu': None,
+                    'restore': None,
+                    'send_chars': None,
+                    'is_maximized': None,
+                    'is_active': None,
+                    'drag_mouse': None,
+                    'get_show_state': None,
+                    'has_style': None,
+                    'user_data': None,
+                    'release_mouse': None,
+                    'move_mouse': None,
+                    'notify_parent': None,
+                    'send_message_timeout': None,
+                    'menu_items': None
+                },
+                # print(pywinauto.controls.hwndwrapper.HwndMeta.str_wrappers)
```

Comment:
Please remove debug prints if they are easy to add again. If they are complicated to remember, please add TODO to remove it in the future.